### PR TITLE
replace prints with kiwi.logger

### PR DESCRIPTION
**Pull request by osab270844**
replace prints with kiwi.logger.

### CODE REVIEW

**Comment by isbm** [comment]
Please resolve the conflicts

**Comment by osab270844** [comment]
> Please resolve the conflicts

Done.

File: src/berry_mill/mill.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def run(self) -> None:
         """
         self._init_local_repos()
         if self.args.show_config:
-            print(yaml.dump(self.cfg.config))
+            log.info(yaml.dump(self.cfg.config))
```
[the remark this answers is not in the document]
Comment (isbm) [reply src/berry_mill/mill.py:158]: @osab270844 any updates here?

[the remark this answers is not in the document]
Comment (osab270844) [reply src/berry_mill/mill.py]: done.

**Comment by isbm** [comment]
> BTW, berrymill expects default configuration at "/etc/berrymill/berrymill.conf", could it be better to check also "/etc/berrymill.conf"?

There are more configs to that, so we keep all in one dir.

**Comment by osab270844** [comment]
> > BTW, berrymill expects default configuration at "/etc/berrymill/berrymill.conf", could it be better to check also "/etc/berrymill.conf"?
> 
> There are more configs to that, so we keep all in one dir.

I see, ok.

**Review by isbm — APPROVED** [review]
lgtm

**Pull request merged**